### PR TITLE
Update docs concerning BMFF-based files

### PR DESCRIPTION
**Pull request by postscript-dev**
Suggested during discussion in #2398.

### CODE REVIEW

File: man/man1/exiv2.1 [hunk 1]
```diff
@@ -76,6 +77,21 @@ image are determined.
 .IP \(bu 2
 Reading other TIFF-like RAW image formats, which are not listed in the
 table, may also work.
+.IP \(bu 2
+Support for BMFF-based types such as AVIF, CR3, HEIF, HEIC and JXL is 
+a build option (see https://github.com/Exiv2/exiv2/blob/main/README.md#BMFF).
```
Comment (kmilos) [inline man/man1/exiv2.1:82]: I'd remove this link as it is transient (or use tag/branch specific address behind e.g. word "here" instead)

File: README.md [hunk 2]
```diff
@@ -49,7 +49,7 @@ The file ReadMe.txt in a build bundle describes how to install the library on th
    16. [Cross Platform Build and Test on Linux for MinGW](#2-16)
    17. [Building with C++11 and other compilers](#2-17)
    18. [Static and Shared Libraries](#2-18)
-   19. [Support for bmff files (CR3, HEIF, HEIC, and AVIF)](#2-19)
+   19. [Support for bmff files (e.g., AVIF, CR3, HEIF, HEIC, and JXL)](#2-19)
```
Comment (kmilos) [inline README.md:52]: No comma after "e.g."?

File: README.md [hunk 3]
```diff
@@ -806,7 +806,7 @@ This is discussed: [https://github.com/Exiv2/exiv2/issues/1230](https://github.c
 [TOC](#TOC)
 <div id="2-19">
 
-### 2.19 Support for bmff files (CR3, HEIF, HEIC, and AVIF)
+### 2.19 Support for bmff files (e.g., AVIF, CR3, HEIF, HEIC, and JXL)
```
Comment (kmilos) [inline README.md:809]: Ditto?

**Comment by kmilos** [comment]
Minor/subtle comment for both PRs: I think I had HEIF, HEIC, AVIF together and in that order because they are sort of a group of same formats: HEIF is the generic one, while HEIC and AVIF indicate what codec is inside...

But it doesn't really matter...

**Comment by postscript-dev** [comment]
@kmilos 
> Minor/subtle comment for both PRs: I think I had HEIF, HEIC, AVIF together and in that order because they are sort of a group of same formats: HEIF is the generic one, while HEIC and AVIF indicate what codec is inside...

My thinking was to list the file types in the same order in all the documentation, as this would make maintenance easier. I copied the order from the manpage. It is a good idea to group the file types as you suggest, so I will make the change.

File: README.md (hunk 2 above)
Comment (postscript-dev) [reply README.md]: > No comma after "e.g."?

I didn't know about the use of [serial commas](https://en.wikipedia.org/wiki/Serial_comma) in US English. In British English, apart from ambiguities, they are usually removed. I will use serial commas in exiv2 docs in the future.

File: man/man1/exiv2.1 (hunk 1 above)
Comment (postscript-dev) [reply man/man1/exiv2.1]: > I'd remove this link as it is transient (or use tag/branch specific address behind e.g. word "here" instead)

Yes, this helps with maintenance.

**Review by kmilos — APPROVED** [review]

**Pull request merged**